### PR TITLE
Implements v2.0.0

### PR DESCRIPTION
**Pull request by roukmoute**

### CODE REVIEW

File: CHANGELOG-2.0.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+This changelog references the relevant changes done in 2.0 version.
+
+### 2.0.0(2018-03-26)
+
+* feature #24583 Adding a new debug:autowiring command (weaverryan)
+* This new version is only for PHP ⩾7.1.
+* Update all dependencies
+* DoctrineParamConverter is not used needed anymore
+* This bundle only convert hashid into an id
```
Comment (Seultou) [inline CHANGELOG-2.0.md]: `convert "s"`

File: CHANGELOG-2.0.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+* Replace `autowire` with `passthrough`.  
+This new parameter which permits to continue with next param converters available.
```
Comment (Seultou) [inline CHANGELOG-2.0.md]: whitespace between `+` and `This` ? or you didn't mean to set a new line ?

File: README.md [hunk 3]
```diff
@@ -75,95 +59,90 @@ $this->get('hashids')->setMinHashLength($minHashLength)->encode(1, 2, 3);
...
 
-Hashids Converter
-=================
+## Hashids Converter
 
 Converter Name: `hashids.converter`
 
-The hashids converter attempts to convert request hashid attributes to a
-id for fetch a Doctrine entity. 
+The hashids converter attempts to convert request hashid attributes to an id.
```
Comment (Seultou) [inline README.md]: `requested` ?

File: CHANGELOG-2.0.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# CHANGELOG for 2.0.x
+
+This changelog references the relevant changes done in 2.0 version.
+
+### 2.0.0(2018-03-26)
+
+* feature #24583 Adding a new debug:autowiring command (weaverryan)
+* This new version is only for PHP ⩾7.1.
+* Update all dependencies
+* DoctrineParamConverter is not used needed anymore
```
Comment (Seultou) [inline CHANGELOG-2.0.md]: `used "and/or" needed anymore` ?

File: CHANGELOG-2.0.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# CHANGELOG for 2.0.x
+
+This changelog references the relevant changes done in 2.0 version.
+
+### 2.0.0(2018-03-26)
+
+* feature #24583 Adding a new debug:autowiring command (weaverryan)
```
Comment (Seultou) [inline CHANGELOG-2.0.md]: `Feature` (caps)

File: README.md [hunk 6]
```diff
@@ -75,95 +59,90 @@ $this->get('hashids')->setMinHashLength($minHashLength)->encode(1, 2, 3);
...
-
-```
-roukmoute_hashids:
-    autowire: true
-```
-
-The autowiring subsystem will detect the hashid.
-
-Base on the example above:
+For specify to use hashids converter just add `"hashid" = "{parameter_name}"`
```
Comment (Seultou) [inline README.md]: `For specific cases, ...` ?

**Comment by Seultou** [comment]
for `declare()` you should put a newline before each one between the php open tag and the `declare(...)` 
[ doc ](https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md#declare-statements-namespace-and-use-declarations)

File: CHANGELOG-2.0.md (hunk 2 above)
Comment (roukmoute) [reply CHANGELOG-2.0.md]: No, it's intended to continue with the previous point.

File: README.md [hunk 7]
```diff
@@ -2,39 +2,44 @@
...
 
-Integrates [hashids/hashids][1] in a Symfony2 project.
+Integrates [hashids/hashids][1] in a Symfony project.
 
-## Installation
+## Installation using composer
 
-Open a command console, enter your project directory and execute the
-following command to download the latest stable version of this bundle:
+Theses commands requires you to have Composer installed globally.
```
Comment (greg0ire) [inline README.md]: "These commands"

File: README.md [hunk 8]
```diff
@@ -75,95 +80,90 @@ $this->get('hashids')->setMinHashLength($minHashLength)->encode(1, 2, 3);
...
-public function getAction(User $user)
+public function getAction(int $user, int $status)
 {
 }
 ```
 
-Or can directly use `id` now !
+## Using Passthrough
+
+`Passthrough` permits to continue with next param converters available.
```
Comment (greg0ire) [inline README.md]: "permits" => "allows"
"the next available param converters"

File: CHANGELOG-2.0.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+This changelog references the relevant changes done in 2.0 version.
+
+### 2.0.0(2018-03-26)
+
+* This new version is only for PHP ⩾7.1.
+* Update all dependencies (remove all doctrine dependencies)
+* DoctrineParamConverter is not used anymore
+* This bundle only converts hashid into an id
+* Replace `autowire` with `passthrough`.  
+This new parameter which permits to continue with the next param converters available.
```
Comment (greg0ire) [inline CHANGELOG-2.0.md]: "allows to continue with the next available param converters"

**Pull request merged**